[Docker] Failed to build

Hi, 

I updated my ort source this morning, and now the build docker is failing.
There is a compilation error :
```
#14 34.57 > Task :downloader:compileKotlin FAILED
#14 34.57 e: /usr/local/src/ort/downloader/src/main/kotlin/vcs/Subversion.kt: (268, 46): Unresolved reference: requestPasswordAuthenticationInstance
#14 34.57
#14 34.57 FAILURE: Build failed with an exception.
```
The method requestPasswordAuthenticationInstance  is not in jdk8 but in the 9.
```
    /**
     * Ask this authenticator for a password.
     *
     * @param host The hostname of the site requesting authentication.
     * @param addr The InetAddress of the site requesting authorization,
     *             or null if not known.
     * @param port the port for the requested connection
     * @param protocol The protocol that's requesting the connection
     *          ({@link java.net.Authenticator#getRequestingProtocol()})
     * @param prompt A prompt string for the user
     * @param scheme The authentication scheme
     * @param url The requesting URL that caused the authentication
     * @param reqType The type (server or proxy) of the entity requesting
     *              authentication.
     *
     * @return The username/password, or null if one can't be gotten
     *
     * @since 9
     */
    public PasswordAuthentication
    requestPasswordAuthenticationInstance(String host,
                                          InetAddress addr,
                                          int port,
                                          String protocol,
                                          String prompt,
                                          String scheme,
                                          URL url,
                                          RequestorType reqType)
```
We have to change `FROM frolvlad/alpine-java:jdk8-slim AS build` to go to a jdk9.
But which one to choose?

--- a/Dockerfile
+++ b/Dockerfile
@@ -17,7 +17,7 @@
 # SPDX-License-Identifier: Apache-2.0
 # License-Filename: LICENSE
 
-FROM frolvlad/alpine-java:jdk8-slim AS build
+FROM adoptopenjdk/openjdk11:jdk-11.0.9_11.1-alpine-slim AS build
 
 # Apk install commands.
 RUN apk add --no-cache \

